themes/survey/fruity_twentythree/views/subviews/content/save.twig

```diff
@@ -186,7 +186,7 @@
                                 'id'          : 'saveemail',
                                 'name'        : 'saveemail',
                                 'class'       : 'form-control ls-important-field',
-                                'placeholder' : gT('enter e-mail address'),
+                                'placeholder' : gT('Enter e-mail address'),
```

Comment:
shouldn't this and the 3 above have the same convention either have your or not first one has your

Reply:
I don't want to change too much until we really have "style guide for copy" (copy = fancy wording for text)